conference/cfp.py

```diff
@@ -51,13 +53,39 @@ def submit_proposal_step1_talk_info(request):
                     request,
                     "Proposal added, now add information about speaker",
                 )
+                send_talk_details_to_backup_email(talk)
                 return redirect("cfp:step2_add_speakers", talk_uuid=talk.uuid)
 
     return TemplateResponse(request, "ep19/bs/cfp/step1_talk_info.html", {
         "proposal_form": proposal_form,
     })
 
 
+def send_talk_details_to_backup_email(talk: Talk):
+    """
+    This is just to double check if we're not loosing any proposals, based on
+    the feedback we've seen on telegram
+    """
+    SEND_CFP_BACKUP_TO = ['web-wg@europython.eu']
+
+    content = f"""
+    title: {talk.title}
+    author: {talk.created_by.id}
+    type_display: {talk.get_type_display()}
+    subtitle: {talk.sub_title},
+    abstract_short: {talk.abstract_short}
+    abstract: {talk.getAbstract().body}
+    abstract_extra: {talk.abstract_extra}
+    """
+
+    send_mail(
+        subject="New Proposal for EP CFP",
```

Comment:
might be worth adding the title in the subject so GMAIL doesn't group all the proposal in one single thread :)